task.go

```diff
@@ -433,6 +433,11 @@ func (t *task) Checkpoint(ctx context.Context, opts ...CheckpointTaskOpts) (Imag
 	if err := t.checkpointTask(ctx, &index, request); err != nil {
 		return nil, err
 	}
+	// jump checkpoint image
+	if len(index.Manifests) == 0 {
```

Comment:
Should we add a check for the criu work path here as well instead of just an empty manifest?

Reply:
Sure, seems checking criu image path is set will be more clearly whether need to jump checkpoint image.